routes/get-current-project.js

```diff
@@ -1,7 +1,10 @@
 var Path = require("path");
 
+var env = require("../lib/environment");
 var utils = require("./utils");
-var DefaultProject = require("../default")()["stay-calm"];
+var defaultProject = require("../default");
+
+var DEFAULT_PROJECT_TITLE = env.get("DEFAULT_PROJECT_TITLE") || "stay-calm";
```

Comment:
can you move this to the config file because this should be consistent for all routes. Then you can reference it in the handler as config.DETAULT_PROJECT_NAME